src/api/MetadataQuery.js

```diff
@@ -133,12 +133,19 @@ class MetadataQuery extends Base {
      */
     flattenResponseEntry = ({ item, metadata }: MetadataQueryResponseEntry): FlattenedMetadataQueryResponseEntry => {
         const { id, name, size } = item;
+        /*
+            - In Content Explorer element, if can_preview permission is false, there is no action taken onClick(item).
+            - /metadata_queries api doesn't yet have permissions returned for each item in the response.
+            - Till the response has permissions, add can_preview: true so that users can click to launch the Preview modal. If users don't have access, they will see the error when preview loads.
```

Comment:
Will add TODO. This will be part of API response soon.Adding this object outside of this file in `ContentExplorer.js` might look out of place and even more confusing. I think we should leave this logic here for time being. Thoughts?